yaksh/views.py

```diff
@@ -117,14 +117,25 @@ def user_logout(request):
 def quizlist_user(request, enrolled=None):
     """Show All Quizzes that is available to logged-in user."""
     user = request.user
-    if enrolled is not None:
+    ci = RequestContext(request)
+
+    if request.method == "POST":
+        course_code = request.POST.get('course_code')
+        hidden_courses = Course.objects.get_hidden_courses(code=course_code)
+        courses = hidden_courses if hidden_courses else None
```

Comment:
Also, could there be a clear way for a student to distinguish between Open courses and hidden courses. With the current implementation, The open courses underneath the search bar would be pretty confusing for the students. Just an heading saying open courses could be enough.